app/components/product/alias-form.hbs

```diff
@@ -0,0 +1,32 @@
+<ul>
+  {{#each @aliases as |alias|}}
+    <li>
+    <div class="flex">
+      <div class="text-sm my-2 text-gray-900">{{alias}}</div>
+      <Rlv::Button
+        @icon="delete-bin-line"
+        @skin="tertiary"
+        @size="small"
+        class="ml-2 my-2 align-middle"
+        {{on "click" (fn this.removeAlias alias)}} />
+    </div>
+    </li>
+  {{/each}}
+  <li>
+    <div class="flex"
+      {{on "focusout" (fn this.addAlias this.newAlias)}}
+    >
+      <Input
+        id="new-alias-name"
+        @value={{this.newAlias}}
+        class="mt-1 focus:ring-red-200 focus:border-red-200 block w-full border-gray-300 rounded-md sm:text-sm" />
+      <Rlv::Button
+        @icon="add-line"
+        @skin="tertiary"
+        @size="small"
+        @disabled={{not this.newAliasIsValid}}
+        class="ml-2 my-1 align-middle"
+        {{on "click" (fn this.addAlias this.newAlias)}} />
```

Comment:
Don't think this button is needed as the aliases is already added on `focusout` of the input. If on focusout `this.newAlias` is valid, I would add that to the array and automatically show a new input-field for another new alias.

Another option is maybe to drop the `focusout` handler and wrap each input-field in a `<form>` with a `submit` button/handler. Then they have the option to click the button or press Enter to add the alias.

Reply:
The form-solution makes a lot of sense. Will do that.